all_test.go

```diff
@@ -268,7 +278,8 @@ func TestResetPassword(t *testing.T) {
 func TestBasicAuth(t *testing.T) {
 	c := setup1(t)
 
-	expect_username_password := func(username, password, expectErrorStart, expectIdentity string) {
+	expect_username_password := func(username, password, expectErrorStart string, userId UserId) {
```

Comment:
Words such as "expect", "truth", "actual" are very important in unit tests. Don't remove such words.